azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/ConfigMojo.java

```diff
@@ -104,30 +92,33 @@ protected boolean confirmConfiguration(WebAppConfiguration configuration) throws
         System.out.println("ResourceGroup : " + configuration.getResourceGroup());
         System.out.println("Region : " + configuration.getRegion());
         System.out.println("PricingTier : " + configuration.getPricingTier());
-        System.out.println("OS : " + configuration.getOs());
-        switch (configuration.getOs()) {
-            case Windows:
-                System.out.println("Java : " + configuration.getJavaVersion());
-                System.out.println("WebContainer : " + configuration.getWebContainer());
-                break;
-            case Linux:
-                System.out.println("RuntimeStack : " + configuration.getRuntimeStack());
-                break;
-            case Docker:
-                System.out.println("Image : " + configuration.getImage());
-                System.out.println("ServerId : " + configuration.getServerId());
-                System.out.println("RegistryUrl : " + configuration.getRegistryUrl());
-                break;
-            default:
-                throw new MojoExecutionException("The value of <os> is unknown.");
+
+        if (configuration.getOs() == null) {
+            System.out.println("No runtime configuration");
```

Comment:
What information do we hope the users to get here?

If the output is only `No runtime configuration`. Then this is much more like a log in debug mode.

Reply:
For we support user didn't config runtime and deploy, and for now we don't know whether user deploy to an existing webapp or just create a new one, so add this information to warn user they didn't complete the runtime if they plan to deploy a new webapp.